parser.js

```diff
@@ -213,6 +214,12 @@ Parser.prototype.parseResults = function(foxyBuffer, callback) {
             gaVisitor.event(ga_params).send();
             callback('weather error');
           });
+      } else if (payload.cmd === FOXY_COMMANDS.MUTE) {
+        ga_params.ea = payload.cmd;
+        gaVisitor.event(ga_params).send();
+
+        shimOptions.body = JSON.stringify(payload);
+        return rp(shimOptions);
```

Comment:
I believe this case should be handled by the "else" case of this if block.  Should be able to remove this code.